fly.toml

```diff
@@ -0,0 +1,30 @@
+# fly.toml app configuration file generated for spokanetech-py on 2024-01-31T20:22:30-08:00
+#
+# See https://fly.io/docs/reference/configuration/ for information about how to use this file.
+#
+
+app = 'spokanetech-py'
+primary_region = 'sea'
+console_command = '/code/manage.py shell'
```

Comment:
I'm not familiar with Fly, but shouldn't this be `/code/src/manage.py`?

Reply:
Good call, yes. This will need fixed. I’ll make a PR